src/validation/obfl.rng

```diff
@@ -23,6 +23,29 @@
       <attribute name="row-spacing"/>
     </optional>
   </define>
+  <define name="translate">
+    <optional>
+      <attribute name="translate">
+        <choice>
+          <value></value>
+          <value>pre-translated</value>
+          <value>uncontracted</value>
+          <value>contracted</value>
+          <data type="string">
+            <param name="pattern">grade:[1-9]+(.[0-9]+)?</param>
+          </data>
+          <value>6-dot</value>
+          <value>8-dot</value>
+          <data type="token">
+            <param name="pattern">(contracted|uncontracted)(/grade:[1-9]+(.[0-9]+)?)?(/([68]-dot))?</param>
```

Comment:
Please add a \ before the .
(in regular expressions, a . stands for any character except new line)